Internal/Debug App/Podfile

```diff
@@ -1,13 +1,13 @@
 # Uncomment the next line to define a global platform for your project
-platform :ios, '10.0'
+platform :ios, '11.0'
```

Comment:
Ideally we should keep the same version as before and bump minimum supported version on the Primer3DS side.

Also we should check the version when initializing 3DS service and throw an error in case merchants are using `2.0.0` of Primer3DS and user is on iOS 10.

Reply:
<img width="1896" alt="Screenshot 2023-08-09 at 11 08 53" src="https://github.com/primer-io/primer-sdk-ios/assets/137198782/73b54f0b-ee3c-41d3-a216-6bd4a32d9ae8">

You can't run pod install because of the minimum iOS version in Podfile.

Reply:
here is the other part of this task https://github.com/primer-io/primer-sdk-3ds-ios/pull/13